integration-tests/build.gradle

```diff
@@ -24,6 +31,13 @@ dependencies {
         exclude module: 'hornetq-native'
     }
 
+    // Import allure-bom to ensure correct versions of all the dependencies are used
+    testImplementation(platform("io.qameta.allure:allure-bom:$versions.allure"))
+    // Add necessary Allure dependencies to dependencies section
+    testImplementation("io.qameta.allure:allure-junit5")
+
+    agent "org.aspectj:aspectjweaver:1.9.21"
```

Comment:
**question:** Do we need to apply aspectJ agent? If we won't use it, do we loose something?

Reply:
According to the [documentation](https://allurereport.org/docs/junit5/#configure-aspectj):
`Allure leverages AspectJ for the functionality of @Step and @Attachment annotations. Additionally, some framework integrations (such as [allure-assertj](https://mvnrepository.com/artifact/io.qameta.allure/allure-assertj)) rely on AspectJ integration to function correctly.`